### ISSUE TITLE
Ensure landing pages have top-level headings

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1058]
A number of the themes lack top-level headings for the landing pages. Themes affected include

- [ ] Thanks Roy
- [ ] Foundation
- [ ] Big Picture
- [ ] The Daily
- [ ] Center Row
- [ ] Seasons
- [ ] Berlin
- [ ] Santa Fe
- [ ] Minimalist
- [ ] Emiglio
- [ ] Freedom

**Comment by kimisgold** [comment]
This is going to be taken care of in the process of resolving #1059.
